bin/release

```diff
@@ -13,6 +13,19 @@ else
   exit 1
 fi
 
+echo "Running the contract test for MI Bridges driving code..."
+RUN_DRIVING_TEST=true bin/rspec spec/features/mi_bridges_driving_spec.rb
```

Comment:
Because we have `set -e` on line 2, I believe if the test fails it will cause the script to stop by default.

Reply:
ok cool, so should I get rid of the following section?

Reply:
I think it's probably fine unless we want to have the browser stay open for a bit so they can do some manual checking?